index.js

```diff
@@ -131,7 +131,7 @@ function bootApp() {
 
   server = require("http-shutdown")(
     app.listen(port, () =>
-      info(`[Metaphysics] Listening on http://localhost:${port}`)
+      info(`[Metaphysics] Listening on http://localhost:${port}/v2`)
```

Comment:
This is #minor, but I feel like the original version is more accurate. The reason is that there are many routes: health check endpoint, root (which is v1), /v2, we apparently support a /graphql route as well, etc. So when the server has booted, its just...ready. All those endpoints are 'listened' on.

But, at the same time, I see what you mean. If you copy + paste the printed out address...that brings you to V1 (in your GraphiQL browser, lets say). That's not good either.

Reply:
I'll follow up with a better log, I think you're point is more important -- one could be confused that the _root_ of the server itself is at `v2`.